nectar/downloaders/threaded.py

```diff
@@ -20,7 +21,7 @@
 DEFAULT_MAX_CONCURRENT = 5
 DEFAULT_BUFFER_SIZE = 8192  # bytes
 DEFAULT_PROGRESS_INTERVAL = 5  # seconds
-DEFAULT_TRIES = 2
+DEFAULT_TRIES = 5
 
```

Comment:
Any specific reason why you decided to increase tries to 5? same for the backoff value?